src/components/Table/GridTable.stories.tsx

```diff
@@ -95,10 +95,14 @@ export function VirtualFiltering() {
   const columns: GridColumn<Row>[] = useMemo(
     () => [
       { header: "Name", data: ({ name }) => name, w: "200px" },
-      { header: "Value", data: ({ value }) => value, w: "10%" },
-      { header: "Value", data: ({ value }) => `${value} `.repeat(10), w: "2fr" },
-      { header: "Value", data: ({ value }) => value },
-      { header: "Action", data: () => <div>Actions</div>, clientSideSort: false },
+      { header: "Value", data: ({ value }) => `${value} `.repeat(10) },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Value", data: ({ value }) => value, w: "200px" },
+      { header: "Action", data: () => <div>Actions</div>, clientSideSort: false, w: "100px" },
```

Comment:
**question:** Did you have any thoughts on the API?

Reply:
No. I still kinda like the `GridColumn.frozen = true`  but would probably do a `GridTable.stickyColumns = { left: N, right: N }` API. Or maybe `GridTable.frozen = { header: true, left: N, right: N }` and remove `stickyHeader`.

Reply:
Maybe all around the term `sticky` might work better since it's supported in CSS?

Reply:
Yeah, but imo sticky is an implementation detail and in discussing with users/PMs/googling for ideas I think this is most often called frozen header/frozen columns, so more tempted to use that term.